autoid_service/autoid.go

```diff
@@ -280,7 +280,7 @@ func New(selfAddr string, etcdAddr []string, store kv.Storage, tlsConfig *tls.Co
 
 func newWithCli(selfAddr string, cli *clientv3.Client, store kv.Storage) *Service {
 	l := owner.NewOwnerManager(context.Background(), cli, "autoid", selfAddr, autoIDLeaderPath)
-	err := l.CampaignOwner()
+	err := l.CampaignOwner(10)
```

Comment:
We can add some commnet here. ```10``` means that autoid service's etcd lease is 10s.

Reply:
Done @hawkingrei